Resources/doc/reference/configuration/cache-manager.rst

```diff
@@ -10,6 +11,7 @@ by default if a :doc:`Proxy Client <proxy-client>` is configured.
     fos_http_cache:
         cache_manager:
             enabled: true
+            custom_proxy_client: ~
```

Comment:
the way the configuration works is that this value defaults to null meaning there is no custom client service. if you configure this to a non-null value, that service will be used instead of the default one.

we can't hardcode the client here, because that depends on whether you configure the symfony or varnish or nginx client.